activerecord/lib/active_record.rb

```diff
@@ -179,6 +178,8 @@ module Tasks
     autoload :SQLiteDatabaseTasks, "active_record/tasks/sqlite_database_tasks"
   end
 
+  Normalization = ActiveModel::Attributes::Normalization
```

Comment:
I'm not sure whether or not this declaration is required, or where it should occur.

Based on [a prior discussion](https://github.com/rails/rails/pull/53887#discussion_r1885364844), there should not be a file to define the `ActiveRecord::Normalization` module.

Since [the `ActiveRecord::Normalization` module exists in the public-facing documentation](https://api.rubyonrails.org/v8.0.0/classes/ActiveRecord/Normalization.html), I'm unsure of this change's responsibility for backwards compatibility. The test suite passes without this line.

Does the _module_ need to continue to exist, or is it more important that the methods themselves continue to exist? If it does need to be preserved, should this commit deprecate it?